docs/using.adoc

```diff
@@ -0,0 +1,311 @@
+command line and it will work as expected.
++
+[NOTE]
+====
+If you encounter a Docker client and server version mismatch such as:
+....
+$ docker ps
+Error response from daemon: client is newer than server (client API version: 1.21, server API version: 1.20)
+....
+You will need to download an earlier compatible version of Docker for
+your host machine. Docker release versions and docker API versions are
+not the same. Typically, you will need to try the previous release (i.e.
+if you get this error message using a docker 1.9 CLI, try a docker 1.8
+CLI).
+====
++
+
+. If you are using Eclipse, you should follow these steps:
+..  Install the
+http://www.eclipse.org/community/eclipse_newsletter/2015/june/article3.php[Docker
+Tooling] plugin.
+,.  Enable the three Docker Views (Docker Explorer, Docker Containers,
+and Docker Images) by choosing **Windows->Show Views->Others**.
+..  Enable the Console by choosing **Windows->Show Views->Console**.
+..  In the `Docker Explorer` view, click to add a connection. You should
+provide a "connection name". If your Environment Variables are set
+correctly, the remaining fields will auto-populate. If not, using the
+output from `vagrant service-manager env docker`, put the DOCKER_HOST
+variable in the "TCP Connection" field and the DOCKER_CERT_PATH in the
+"Authentication Section" Path.
+..  You can test the connection and then accept the results. At this
+point, you are ready to use ADB with Eclipse.
+
+NOTE: Testing has been done with Eclipse 4.5.0.
+
+[[using-the-box-via-ssh]]
+== Using the box via SSH
+
+Today, most users will work inside the Vagrant box. Access the box by
+using `ssh` to login to it with the following command:
+
+....
+vagrant ssh
+....
+
+You are now at a shell prompt inside the Vagrant box. You can now
+execute commands and use the tools provided.
+
+You can use the
+https://github.com/projectatomic/adb-utils/blob/master/README.rst[sccli]
+to manage the orchestration services inside of ADB. `sccli` makes it
+easy to start and stop orchestration providers like Kubernetes or
+OpenShift.
+
+[[using-docker]]
+=== Using `docker`
+
+ADB provides a full container environment and runs both `docker` and
+`kubernetes`. All standard commands work, for example:
+
+....
+ADB pull centos
+ docker run -t -i centos /bin/bash
+....
+
+[[using-atomic-app-and-nulecule]]
+=== Using Atomic App and Nulecule
+
+Details on these projects can be found at these URLs:
+
+* Atomic App: https://github.com/projectatomic/atomicapp
+* Nulecule: https://github.com/projectatomic/nulecule
+
+The
+https://registry.hub.docker.com/u/projectatomic/helloapache/[helloapache]
+example can be used to test your installation.
+
+NOTE: Many Nulecule examples expect a working Kubernetes environment.
+Use the
+link:../components/centos/centos-k8s-singlenode-setup/Vagrantfile[Vagrantfile]
+and refer the corresponding
+link:../components/centos/centos-k8s-singlenode-setup/README.rst[README]
```

Comment:
.rst>.adoc.
Maybe once this whole exercise is complete we can use https://wummel.github.io/linkchecker/ to check for broken links. I have not used it but it was highly recommended by one of the developers.